AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/AppLovin/Banners/UI/ALDemoSwiftUIBannerView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ALDemoSwiftUIBannerView.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Matthew Nguyen on 7/31/23.
+//  Copyright © 2023 AppLovin. All rights reserved.
+//
+
+import SwiftUI
+import Adjust
+import AppLovinSDK
+
+@available(iOS 13.0, *)
+struct ALDemoSwiftUIBannerView: View
+{
+    @ObservedObject private var bannerViewModel = ALDemoAdViewSwiftUIViewModel()
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            callbacksTable
+                .frame(maxHeight: .infinity)
+            
+            ALAdViewSwiftUIWrapper(shouldLoadAd: $bannerViewModel.shouldLoadAd,
+                                   adLoaded: $bannerViewModel.adLoaded,
+                                   adFormat: .banner,
+                                   didLoad: bannerViewModel.adService(_:didLoad:),
+                                   didFailToLoadAdWithError: bannerViewModel.adService(_:didFailToLoadAdWithError:),
+                                   wasDisplayedIn: bannerViewModel.ad(_:wasDisplayedIn:),
+                                   wasHiddenIn: bannerViewModel.ad(_:wasHiddenIn:),
+                                   wasClickedIn: bannerViewModel.ad(_:wasClickedIn:),
+                                   didPresentfullscreenFor: bannerViewModel.ad(_:didPresentFullscreenFor:),
+                                   willDismissFullscreenFor: bannerViewModel.ad(_:willDismissFullscreenFor:),
+                                   didDismissFullscreenFor: bannerViewModel.ad(_:didDismissFullscreenFor:),
+                                   willLeaveApplicationFor: bannerViewModel.ad(_:willLeaveApplicationFor:),
+                                   didReturnToApplicationFor: bannerViewModel.ad(_:didReturnToApplicationFor:),
+                                   didFailToDisplayIn: bannerViewModel.ad(_:didFailToDisplayIn:withError:))
+                .deviceSpecificFrame()
+            
+            Button {
+                bannerViewModel.shouldLoadAd = true
+            } label: {
+                Text("Load")
+            }
+                .disabled(bannerViewModel.shouldLoadAd)
+                .padding(.top)
+        }
+            .background(Color.black.edgesIgnoringSafeArea(.all))
+    }
```

Comment:
Could we also add the title to the navigation bar (as it appears in our storyboard view controllers), e.g., "SwiftUI Banners" here?

Reply:
I was able to add the title to the navigation bar through code by setting the title inside a `NavigationView`, but the `.navigationTitle()` modifier on the NavigationView is only supported on iOS 14.0+, so I had to change the requirements on the ad views from `@available(iOS 13.0, *)` to `@available(iOS 14.0, *)` Let me know if that's fine!

Reply:
The way to do it on iOS 13 is using `navigationBarTitle()` with the view embedded in a `NavigationView` but it was deprecated so I'd say it's fine.